src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -7,12 +7,13 @@
 package org.gridsuite.study.server;
 
 import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowResult;
 import io.swagger.annotations.*;
 import java.nio.charset.StandardCharsets;
 import java.util.*;
 import java.util.logging.Level;
 import org.gridsuite.study.server.dto.*;
-import org.gridsuite.study.server.repository.StudyEntity;
+import org.gridsuite.study.server.entities.StudyEntity;
```

Comment:
Why did you change the package ? It makes it harder to review the migration

Reply:
the packages contains entities and not  repositories, there's another package that contains repositories